File: JAVA/01_core_JAVA/Lecture02/pattern5.java
//Code-Make a cross
package Lecture02;

import java.util.Scanner;

public class pattern5 {

	public static void main(String[] args) {
		Scanner scn = new Scanner(System.in);
		System.out.print("Enter odd number of rows :");
		int n = scn.nextInt();
		for (int i = 0; i < n; i++) {
			for (int j = 0; j < n; j++) {
				// check the available position in square
				if (i == j || (i + j) == n - 1) {
					System.out.print("*");
				} else {
					System.out.print(" ");
				}
			}
			System.out.println();
		}

	}

}

File: C++ New/01_Basic/Lec_02/pattern5.cpp
//code- print a cross "It is valid for odd number of rows"
#include <iostream>
using namespace std;
int main(){
    int row;
    cout << "Enter the number of rows : ";
    cin >> row;
    for(int i = 0; i < row; i++){
        for (int j = 0; j < row; j++){
            if (i == j || (i + j) == row - 1){
                cout << "*";
            } else {
                cout << " ";
            }
        }
        cout << endl;
    } 
    return 0;
}